src/cpuinfo_x86.c

```diff
@@ -1330,6 +1331,43 @@ static void ParseCpuId(const uint32_t max_cpuid_leaf, X86Info* info,
     features->ssse3 = GetDarwinSysCtlByName("hw.optional.supplementalsse3");
     features->sse4_1 = GetDarwinSysCtlByName("hw.optional.sse4_1");
     features->sse4_2 = GetDarwinSysCtlByName("hw.optional.sse4_2");
+#elif defined(CPU_FEATURES_OS_FREEBSD)
+    // Handling FreeBSD platform through parsing /var/run/dmesg.boot.
+    const int fd = CpuFeatures_OpenFile("/var/run/dmesg.boot");
+    if (fd >= 0) {
+      StackLineReader reader;
+      StackLineReader_Initialize(&reader, fd);
+      for (;;) {
+        const LineResult result = StackLineReader_NextLine(&reader);
+        const StringView line = result.line;
+        const bool is_feature =
+            CpuFeatures_StringView_StartsWith(line, str("  Features="));
+        const bool is_feature2 =
+            CpuFeatures_StringView_StartsWith(line, str("  Features2="));
+        if (is_feature || is_feature2) {
+          // Lines of interests are of the following form:
+          // "  Features=0x1783fbff<PSE36,MMX,FXSR,SSE,SSE2,HTT>"
+          // We replace '<', '>' and ',' with space so we can search by
+          // whitespace separated word.
+          for (size_t i = 0; i < line.size; ++i) {
+            if (line.ptr[i] == '<' || line.ptr[i] == '>' || line.ptr[i] == ',')
+              line.ptr[i] == ' ';
```

Comment:
use of `==` instead of `=`
`line.ptr[i] == ' ';` -> `line.ptr[i] = ' ';`

ps: Thanks warning !

Reply:
Thx! I should have read your comment earlier, I just pushed a few broken commits XD.